tool/teleport/common/teleport.go

```diff
@@ -299,6 +299,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	dump.Flag("auth-server", "Address of the auth server.").StringVar(&dumpFlags.AuthServer)
 	dump.Flag("app-name", "Name of the application to start when using app role.").StringVar(&dumpFlags.AppName)
 	dump.Flag("app-uri", "Internal address of the application to proxy.").StringVar(&dumpFlags.AppURI)
+	dump.Flag("node-labels", "Comma-separated list of labels to add to newly created nodes ex) env:staging,cloud:aws.").StringVar(&dumpFlags.NodeLabels)
```

Comment:
The UI asks the user to input the labels as a `key:value` pair and also displays them as such but using this format doesn't jive with the `ParseLabelSpec` method you mention below. I feel like we should be consistent with the formatting everywhere and so maybe this should be updated to use the colon delimiter instead of the other way around?

Reply:
My 2c:

The CLI for `teleport configure` should be consistent with any other teleport CLI.

If that differs from the web UI, then the web UI should take care of the conversion.